### DISCUSSION TITLE
how do i know wheather the inline completions were showed?

### DISCUSSION THREAD

**Discussion by hanyl2022** [microsoft/vscode-discussions#1168] [Extension Development QnA]
I am developing a plugin that can give some tips with inline completion function.
I found than the inline completions were not shown by vscode. I wonder whether there is a way to know my content was shown. 
Is there any event i can capture?

**Comment by isidorn** [comment]
What's your use case for knowing this?
If VS Code calls your inline completion provider, then the first inline completion you return will be shown

**Comment by abrilurena** [comment]
Hi, @hanyl2022! hope you're doing great. Do you still need help with this?

**Comment by burkeholland** [comment]
@hanyl2022 2nd check for an update here. If you have one, please update this discussion. If this item is still open during our next moderation pass, we'll close it. You can always re-open if it's still an issue. Thanks!
